test/remote/gateways/remote_cyber_source_rest_test.rb

```diff
@@ -0,0 +1,82 @@
+require 'test_helper'
+
+class RemoteCyberSourceRestTest < Test::Unit::TestCase
+  def setup
+    @gateway = CyberSourceRestGateway.new(fixtures(:cybersource_rest))
+    @amount = 10221
+    @card_with_out_funds = credit_card('42423482938483873')
```

Comment:
Maybe my smallest nitpick ever
```suggestion
    @card_without_funds = credit_card('42423482938483873')
```